CHANGELOG.md

```diff
@@ -6,6 +6,20 @@
 
 ### Fixes
 
+## v1.21.0 (2022-03-06)
```

Comment:
I noticed the date format changed from `MMM DD, YYYY` to `YYYY-MM-DD`, so following the convention from last release.

cc @yurishkuro